README.md

```diff
@@ -5,7 +5,7 @@ There is a list of all tools and dependencies required for this tutorial.
 [rustup](https://github.com/rust-lang-nursery/rustup.rs#installation) is the easiest way to install Rust toolchains. Rust nightly toolchain is required since our contracts require some unstable features:
 
 ```bash
-rustup install nightly
+rustup install nightly-2018-06-28
```

Comment:
If you leave it as it is then you make user download *two* toolchains. Just have exactly this situation: I install a toolchain (which takes a while), when I run `./buils.sh` which starts to download yet another toolchain (which is specified in rust-toolchain file). Beside that, if user installs `wasm` toolchain on `nightly` it won't be available on `nightly-whatever-date` because it require separate installation.

Reply:
if so, isn't it better to just remove this instruction from the tutorial? My point is following: toolchain upgrade will take place periodically and it's easy to forget to update it in different places, so it's better to avoid it if we could.

Reply:
Well, I think it doesn't matter if you change 7 or 8 places. But it makes clearer which exact toolchain user should have in order to run all steps. Otherwise it would look like "run `build.sh`. Wait until installation finishes with error. add `wasm32` target. Now run `build.sh` again. It should complete without errors". Do you think it's better than just keep it in readme? All toolchain versions should be updated simultaneously, e.g. via replace all, which won't forget to change readme too.

Reply:
fair points.